extscheduler/resources/schemas/dbscripts/sqlserver/extscheduler-21.410-21.411.sql

```diff
@@ -0,0 +1,147 @@
+USE [Labkey_test]
```

Comment:
Please remove the USE statement here. This will prevent it from working on production which uses a different DB name.